cleanrepo/CleanRepo/Program.cs

```diff
@@ -1591,6 +1718,67 @@ private static List<FileInfo> GetYAMLFiles(string directoryPath)
 
         return mediaFiles;
     }
+    /// <summary>
+    /// Returns a dictionary of all .png/.jpg/.gif/.svg files in the directory.
+    /// The search includes the text found in the files.
+    /// </summary>
+    private static Dictionary<string, string> ScanMediaFiles(List<string>? imageFilePaths, string ocrModelDirectory)
+    {
+
+        Dictionary<string, string> ocrDataForFiles = new(StringComparer.InvariantCultureIgnoreCase);
+
+        if (imageFilePaths is null or { Count : 0 })
+        {
+            Console.WriteLine("\nNo .png/.jpg/.gif/.svg files to scan!");
+            return ocrDataForFiles;
+        }
+
+        using (var engine = new TesseractEngine(ocrModelDirectory, "eng", EngineMode.Default))
+        {
+            foreach (string imageFilePath in imageFilePaths)
+            {
+                using (var img = Pix.LoadFromFile(imageFilePath))
+                {
+                    using (Page page = engine.Process(img))
+                    {
+                        string text = page.GetText();
+                        ocrDataForFiles.Add(imageFilePath, text);
+                    }
+                }
+            }
+        }
```

Comment:
This could be simplified with an updated syntax for a `using` statement:

```suggestion
        using var engine = new TesseractEngine(ocrModelDirectory, "eng", EngineMode.Default);
        foreach (string imageFilePath in imageFilePaths)
        {
            using var img = Pix.LoadFromFile(imageFilePath);
            using Page page = engine.Process(img);

            string text = page.GetText();
            ocrDataForFiles.Add(imageFilePath, text);
        }
```